tests/main.c

```diff
@@ -63,8 +66,10 @@ int main(int argc, char **argv) {
         DECLARE_TEST(Aarch64MinGW, TestVaList),
         DECLARE_TEST(Aarch64MinGW, TestSPrintf),
         DECLARE_TEST(Aarch64MinGW, TestVaArgPack),
+        DECLARE_TEST(Aarch64MinGW, UnwindStackTest),
         DECLARE_TEST(Aarch64MinGW, WeakReferencesTest)
     };
 
+    setvbuf(stdout, NULL, _IONBF, 0);
```

Comment:
What is this doing?

Reply:
It disables buffering that was important for SEH tests.

Reply:
Thank you for the explanation. I'd add a comment with that.

Reply:
Added.